mod/groups/start.php

```diff
@@ -72,7 +72,6 @@ function groups_init() {
 	elgg_register_plugin_hook_handler('prepare', 'menu:title', '_groups_title_menu_prepare');
 
 	elgg_register_plugin_hook_handler('gatekeeper', 'group:group', '_groups_gatekeeper_allow_profile_page');
-	elgg_register_plugin_hook_handler('forward', '403', '_groups_gatekeeper_redirect');
```

Comment:
was this no longer needed?

Reply:
Nope. This PR simplifies forwarding, so you can set the forward URL when you throw the exception. Will need to update Gatekeeper exceptions the same way

Reply:
i overlooked it being set in the exception... https://github.com/Elgg/Elgg/pull/11802/files#diff-27d4f8bc1435decc9f0da4955fb84c29R262